### PR TITLE
Update Pull Request template with checklist

### LINKED ISSUE #8045: Update PR Template with Checklist
I updated the Issue Template with #7958 and I feel the Pull Request Template could use an update too. [This website can autogenerate a template](https://www.talater.com/open-source-templates) by asking you a series of questions on what you need in a PR and it'll generate a template for you. Here's the template I got:

```
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Screenshots (if appropriate):

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
```

The reason I am for checklists for PRs is because we ask things like if you've tested locally or not. Rather than type it every time, you can just add an `x` to cross it off.

### New Template

I propose the following as the new template. ~~I've removed a lot of instructions from the current template as Camperbot takes care of a lot of them, like make sure you only have one commit.~~ Any feedback is welcome.

```
<!-- FreeCodeCamp Pull Request Template -->

<!-- IMPORTANT Please review the https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md for detailed contributing guidelines -->
<!-- Make sure that you PR is not a duplicate -->

## Pre-Submission Checklist
<!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!-- All of these points should be checked, otherwise, read the CONTRIBUTING guidelines from above-->
<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] Branches much start with either `fix/` or `feature/`
- [ ] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/         FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [ ] All new and existing tests pass `npm run test-challenges`.

## Description
<!-- Describe your changes in detail -->

## Types of changes
<!-- What types of changes does your code introduce? Put one `x` in the box that applies. -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Add new translation (feature adding new translations)

## Checklist:
<!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] Tested changes locally.
- [ ] Closes currently open issue (`Closes #XXXX`): Closes 
```

### New Template Rendered

> 
> 
> 
> 
> 
> ## Pre-Submission Checklist
> 
> 
> 
> - [ ] Branches much start with either `fix/` or `feature/`
> - [ ] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/         FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
> - [ ] All new and existing tests pass `npm run test-challenges`.
> 
> ## Description
> 
> 
> ## Types of changes
> 
> - [ ] Bug fix (non-breaking change which fixes an issue)
> - [ ] New feature (non-breaking change which adds functionality)
> - [ ] Breaking change (fix or feature that would cause existing functionality to change)
> - [ ] Add new translation (feature adding new translations)
> 
> ## Checklist:
> 
> 
> - [ ] Tested changes locally.
> - [ ] Closes currently open issue (`Closes #XXXX`): Closes

### PR DESCRIPTION
**Pull request by erictleung**
Tested through GitHub preview. Closes #8045.

Plain Text:

```
<!-- FreeCodeCamp Pull Request Template -->

<!-- IMPORTANT Please review https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md for detailed contributing guidelines -->
<!-- Help with PRs can be found at https://gitter.im/FreeCodeCamp/HelpContributors -->
<!-- Make sure that your PR is not a duplicate -->

#### Pre-Submission Checklist
<!-- Go over all points below, and put an `x` in all the boxes that apply. -->
<!-- All points should be checked, otherwise, read the CONTRIBUTING guidelines from above-->
<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] Your pull request targets the `staging` branch of FreeCodeCamp.
- [ ] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [ ] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [ ] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.

#### Type of Change
<!-- What type of change does your code introduce? Put an `x` in the box that applies. -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Add new translation (feature adding new translations)

#### Checklist:
<!-- Go over all points below, and put an `x` in all the boxes that apply. -->
<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] Tested changes locally.
- [ ] Closes currently open issue (`Closes #XXXX`): Closes 

#### Description
<!-- Describe your changes in detail -->
```

Rendered:

> 
> 
> 
> 
> 
> 
> 
> #### Pre-Submission Checklist
> 
> 
> 
> 
> 
> 
> - [ ] Your pull request targets the `staging` branch of FreeCodeCamp.
> - [ ] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
> - [ ] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
> - [ ] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
> #### Type of Change
> 
> 
> - [ ] Bug fix (non-breaking change which fixes an issue)
> - [ ] New feature (non-breaking change which adds functionality)
> - [ ] Breaking change (fix or feature that would cause existing functionality to change)
> - [ ] Add new translation (feature adding new translations)
> #### Checklist:
> 
> 
> 
> 
> - [ ] Tested changes locally.
> - [ ] Closes currently open issue (`Closes #XXXX`): Closes 
> #### Description
> 
>

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
-Please, go through these steps before you submit a PR.
+<!-- FreeCodeCamp Pull Request Template -->
+
+<!-- IMPORTANT Please review https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md for detailed contributing guidelines -->
+<!-- Make sure that your PR is not a duplicate -->
+
+#### Pre-Submission Checklist
+<!-- Go over all points below, and put an `x` in all the boxes that apply. -->
+<!-- All points should be checked, otherwise, read the CONTRIBUTING guidelines from above-->
+<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
```
Comment (BerkeleyTrue) [inline .github/PULL_REQUEST_TEMPLATE.md]: What do you think about adding a link to the helpContributors room here?

Comment (erictleung) [inline .github/PULL_REQUEST_TEMPLATE.md]: Good idea! The issue template has one so it's fitting to have one of the Gitter rooms in the PR template too.

Comment (raisedadead) [inline .github/PULL_REQUEST_TEMPLATE.md]: Umm @erictleung You want to add that link?

Comment (erictleung) [inline .github/PULL_REQUEST_TEMPLATE.md]: @raisedadead I'm not sure why it doesn't show up above but it is added if you look at the file 😄

Comment (raisedadead) [inline .github/PULL_REQUEST_TEMPLATE.md]: Cool.

**Pull request merged**